xsd/netex_framework/netex_reusableComponents/netex_facility_support.xsd

```diff
@@ -872,12 +888,37 @@ Rail transport, Roads and Road transport
 			<xsd:restriction base="FacilitySetRefStructure">
 				<xsd:attribute name="ref" type="ServiceFacilitySetIdType" use="required">
 					<xsd:annotation>
-						<xsd:documentation>Identifier of a FACILITY.</xsd:documentation>
+						<xsd:documentation>Identifier of a FACILITY SET.</xsd:documentation>
 					</xsd:annotation>
 				</xsd:attribute>
 			</xsd:restriction>
 		</xsd:simpleContent>
 	</xsd:complexType>
+	<xsd:element name="RestrictedServiceFacilitySetRef" type="RestrictedServiceFacilitySetRefStructure">
+		<xsd:annotation>
+			<xsd:documentation>Reference to a restricted SERVICE FACILITY SET. The restriction can be an AvailabiltyCondition and/or a from - to relation. +V2.0</xsd:documentation>
+		</xsd:annotation>
+	</xsd:element>
+	<xsd:complexType name="RestrictedServiceFacilitySetRefStructure">
+		<xsd:annotation>
+			<xsd:documentation>Reference to a restricted SERVICE FACILITY SET. The restriction can be an AvailabiltyCondition and/or a from - to relation. +V2.0</xsd:documentation>
+		</xsd:annotation>
+		<xsd:sequence>
+			<xsd:element ref="ServiceFacilitySetRef"/>
+			<xsd:choice minOccurs="0">
+				<xsd:sequence>
+					<xsd:element name="FromCall" type="CallRefStructure"/>
+					<xsd:element name="ToCall" type="CallRefStructure"/>
+				</xsd:sequence>
+				<xsd:sequence>
+					<xsd:element name="FromStopPointInJourneyPattern" type="StopPointInJourneyPatternRefStructure"/>
+					<xsd:element name="ToStopPointInJourneyPattern" type="StopPointInJourneyPatternRefStructure"/>
+				</xsd:sequence>
+			</xsd:choice>
+			<xsd:element ref="AvailabilityConditionRef" minOccurs="0">
```

Comment:
This AvailabilityCondition overrides the one from the referenced ServiceFacilitySet (if it has one), right ?

Reply:
Why would we use a StopPointInJourneyPattern here, while we use Stop directly in the JourneyPart?

Reply:
Yes, it overrides it (should add this in the description.  Because I thought that the StopPointInSP is the  equivalent to the call.